guides/common/modules/proc_configuring-projectserver-to-use-cross-forest-trust.adoc

```diff
@@ -0,0 +1,64 @@
+[id="Configuring-a-cross-forest-trust-between-{FreeIPA-context}-and-Active-Directory-for-{Project}_{context}"]
+= Configuring a cross-forest trust between {FreeIPA} and Active Directory for {Project}
+
+When a cross-forest trust is configured in your {FreeIPA} deployment, configure host-based access control (HBAC) and the System Security Services Daemon (SSSD) to enable Active Directory users to log in to {Project}.
```

Comment:
Hm, not quite. I see configuring a trust as part of the actual FreeIPA installation; and, by analogy, it would feel weird to write "If you install FreeIPA". But I think this might address your concerns as well:

```suggestion
When your {FreeIPA} deployment includes a cross-forest trust with Active Directory, configure host-based access control (HBAC) and the System Security Services Daemon (SSSD) to enable Active Directory users to log in to {Project}.
```

Reply:
Yeah, that works.